bundles/org.openhab.ui.dashboard/web/js/geolocation.js

```diff
@@ -35,7 +37,7 @@
 
     function send(configuration) {
         $.ajax({
-            url : '../rest/services/org.eclipse.smarthome.core.i18nprovider/config',
+            url : window.location.origin + '/rest/services/org.eclipse.smarthome.core.i18nprovider/config',
```

Comment:
Relative path wouldn't let me test it from `http://openhab:8080/static/` page. This approach is safer. Let me know if there are any obstacles.